interface/controller/student_controller.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"encoding/csv"
+	"errors"
+	"golang-bootcamp-2020/domain/model"
+	"io"
+	"log"
+	"os"
+	"strconv"
+)
+
+type StudentController interface {
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name